doc/managing-wallets.md

```diff
@@ -108,7 +108,7 @@ Wallets created before version 0.13 are not HD and must be backed up every 100 k
 
 ### 1.6 Restoring the Wallet From a Backup
 
-To restore a wallet, the `restorewallet` RPC must be used.
+To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item (`File` -> `Restore Wallet...`) must be used.
```

Comment:
nit: Actually, the symbol is [ellipsis](https://en.wikipedia.org/wiki/Ellipsis):
```suggestion
To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item (`File` -> `Restore Wallet…`) must be used.
```

See https://github.com/bitcoin/bitcoin/blob/edc3d1b296e34838d649dc21b8483a52e214932a/src/qt/bitcoingui.cpp#L353

Reply:
Thanks, @hebasto. I'll change the "`...`" usage to "`…`" at the end of line 91 as well.

https://github.com/bitcoin/bitcoin/blob/118ec94d70b293b95aa9d2713733c8e1d9b63b92/doc/managing-wallets.md?plain=1#L91